src/database/models/vacationRequest.ts

```diff
@@ -3,35 +3,35 @@
  * DynamoDB model for vacation request
  */
 interface VacationRequestModel {
-    id: string;
-    personId: string;
-    draft: boolean;
-    startDate: string;
-    endDate: string;
-    days: number;
-    type: VacationType;
-    status: VacationRequestStatuses;
-    message: string;
-    createdBy: string;
-    createdAt: string;
-    updatedAt: string;
-    updatedBy: string;
+  id: string;
+  personId: string;
+  draft: boolean;
+  startDate: string;
+  endDate: string;
+  days: number;
+  type: VacationType;
+  status: VacationRequestStatuses;
+  message: string;
+  createdBy: string;
+  createdAt: string;
+  updatedAt: string;
+  updatedBy: string;
 }
 
 /**
  * Enumerator for the vacation request statuses
  */
 enum VacationRequestStatuses {
-    PENDING = "PENDING",
-    APPROVED = "APPROVED",
-    REJECTED = "DECLINED"
+  PENDING = "PENDING",
+  APPROVED = "APPROVED",
+  REJECTED = "DECLINED"
 }
```

Comment:
This is not currently in use. Also when this is needed/ in use we can use the enum which is already provided from the generated files